worker/include/RTC/RtpStreamSend.hpp

```diff
@@ -36,23 +36,43 @@ namespace RTC
 			bool rtxEncoded{ false };
 		};
 
+		class Bucket {
```

Comment:
This seems to be a good candidate to be a template class accessible as an utility. In this concreate case it woul dbe a `Bucket` of `StorageItem`.

Reply:
I'll prepare commit only for ```Bucket<T>```

Reply:
Moved into ```Utils.hpp```